CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Batch of package names reservation
```

Comment:
I prefer something more to-the-point for the user like:
```
- `batch_upload_to_pypi` method added to `Uploader`
```

Reply:
ok